cypress/e2e/functional/document_tests/tiles_copy_test_spec.js

```diff
@@ -302,3 +307,46 @@ context('Test copy tiles from one document to other document', function () {
 
   });
 });
+
+context("Test copy tile within a document", function () {
+  it("Copies a graph tile within a document", function () {
+    beforeTest(student5);
+
+    // Add table tile and populate it with categorical data.
+    cy.log("Add table tile with categorical data");
+    clueCanvas.addTile("table");
+    cy.get(".primary-workspace").within((workspace) => {
+      tableToolTile.typeInTableCellXY(0, 0, "small");
+      tableToolTile.getTableCellXY(0, 0).should("contain", "small");
+      tableToolTile.typeInTableCellXY(1, 0, "medium");
+      tableToolTile.getTableCellXY(1, 0).should("contain", "medium");
+      tableToolTile.typeInTableCellXY(0, 1, "red");
+      tableToolTile.getTableCellXY(0, 1).should("contain", "red");
+      tableToolTile.typeInTableCellXY(1, 1, "green");
+      tableToolTile.getTableCellXY(1, 1).should("contain", "green");
+    });
+
+    // Graph the table data in a new graph tile
+    cy.get("[data-original-title='Graph It!']").click();
+    cy.get("[data-test=link-tile-select]").select("New Graph");
+    cy.get(".modal-button").contains("Graph It").click();
+    cy.get(".primary-workspace .graph-wrapper").should("have.length", 1);
```

Comment:
These are fine, but there are also helper methods for some of these selectors, eg `XYPlotToolTile.getTile()` which might be more readable.